utils/build/test.mk

```
@@ -0,0 +1,92 @@
+
+# This is like build.mk but for test files
+T_CFLAGS  := $(CFLAGS)
+T_LFLAGS  := $(LFLAGS)
+T_INC_DIR := $(INC_DIR)
+T_TARGET  := testrunner
```

This currently regenerates testrunner in root directory. 

```suggestion
T_TARGET  := $(TEST_DIR)/testrunner
```

---

That was also the original way of running the test. Let's hope moving this won't break the scripts running the testrunner.